### PR TITLE
[v9.5.x] docs: update email sharing feature info

### PR DESCRIPTION
**Pull request by imatwawana**
Removes feature flag information for email sharing
Updates availability notes
Fixes typos

### CODE REVIEW

File: docs/sources/dashboards/dashboard-public/index.md [hunk 1]
```diff
@@ -220,14 +217,14 @@ guaranteed because plugin developers can override this functionality. The follow
 
 ## Limitations
 
-- Panels that use frontend datasources will fail to fetch data.
+- Panels that use frontend datas ources will fail to fetch data.
```
Comment (imatwawana) [inline docs/sources/dashboards/dashboard-public/index.md:220]:
```suggestion
- Panels that use frontend data sources will fail to fetch data.
```
```suggestion
- Panels that use frontend data sources will fail to fetch data.
```

**Review by thanos-karachalios — COMMENTED** [review]
All changes seem to be ok. The only point would be around be clear around pricing since the G10 docs will mention that.

File: docs/sources/dashboards/dashboard-public/index.md [hunk 2]
```diff
@@ -66,20 +70,13 @@ The link no longer works. You must create a new public URL as in [Make a dashboa
...
+Available in public preview in [Grafana Cloud Pro and Advanced](/docs/grafana-cloud). Please contact support to have the feature enabled.
 
-Add the `publicDashboardsEmailSharing` feature toggle to your `custom.ini` file.
+{{% /admonition %}}
 
-```
-[feature_toggles]
-publicDashboardsEmailSharing = true
-```
+Email sharing allows you to share your public dashboard with only specific people via email, instead of having it accessible to anyone with the URL.
```
[the remark this answers is not in the document]
Comment (imatwawana) [reply docs/sources/dashboards/dashboard-public/index.md:79]: The pricing information isn't in the docs, just in the What's new. But we can add in both doc sets if you want it there.

File: docs/sources/dashboards/dashboard-public/index.md [hunk 3]
```diff
@@ -8,9 +8,13 @@ weight: 8
 
 # Public dashboards
 
-> **Note:** This is an opt-in alpha feature.
+{{% admonition type="note" %}}
 
-> **Caution:** Making your dashboard public could result in a large number of queries to the datasources used by your dashboard.
+This feature is in public preview.
```
Comment (imatwawana) [inline docs/sources/dashboards/dashboard-public/index.md:13]:
```suggestion
This feature is experimental.
```

File: docs/sources/dashboards/dashboard-public/index.md [hunk 4]
```diff
@@ -66,20 +70,13 @@ The link no longer works. You must create a new public URL as in [Make a dashboa
...
 
-> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud).
-
-Email sharing allows you to share your public dashboard with only specific people via email, instead of having it accessible to anyone with the URL.
-
-### Enable email sharing
+{{% admonition type="note" %}}
 
-> **Note:** For Grafana Cloud, contact support to have the feature enabled.
+Available in public preview in [Grafana Cloud Pro and Advanced](/docs/grafana-cloud). Please contact support to have the feature enabled.
```
Comment (imatwawana) [inline docs/sources/dashboards/dashboard-public/index.md:75]:
```suggestion
This feature is experimental in [Grafana Cloud Pro and Advanced](/docs/grafana-cloud). Please contact support to have the feature enabled. This feature will have a cost by active users after being promoted into general availability.
```

**Comment by imatwawana** [comment]
If this looks good to you now @thanos-karachalios, can you please approve the PR. I've also opened PR #69206 to add the pricing information to the V10 docs.

**Review by evictorero — APPROVED** [review]

**Pull request merged**